benches/macrobenches.rs

```diff
@@ -1,6 +1,6 @@
 use criterion::{self, criterion_group, criterion_main, Criterion, Throughput};
 use quick_xml::events::Event;
-use quick_xml::Reader;
```

Comment:
Let's at least make them available in a `prelude::*`

Reply:
Do you think that this will cause usability issues? I don't think that it is needed to get this for that -- then it is just better to drop that commit.

Reply:
I'd prefer to keep everything available from the root namespace, but you don't have to drop the entire commit.  It still makes sense to have `reader` and `writer` modules.

Reply:
Ok, therefore, because after my updates it does not add any value to async PR I'll move it to another small PR with various stuff